### PR TITLE
Azure Synapse Analytics Dedicated dialect support

### PR DESCRIPTION
**Pull request by alex-odysseus**
Addresses https://github.com/OHDSI/WebAPI/issues/2071

To be aligned with https://github.com/OHDSI/WebAPI/pull/2073

It can be merged only after releasing of the corresponding libraries:
- DatabaseConnector (@schuemie please support to merge https://github.com/odysseusinc/DatabaseConnector/tree/synapse into https://github.com/OHDSI/DatabaseConnector/tree/develop and release after)
- SqlRender (@schuemie please support to merge https://github.com/odysseusinc/SqlRender/tree/synapse into https://github.com/OHDSI/SqlRender/tree/develop and release after)
- CommonDataModel (@clairblacketer please support to merge https://github.com/odysseusinc/CommonDataModel/tree/synapse into https://github.com/OHDSI/CommonDataModel/tree/main)
- ARACHNE Commons (https://github.com/OHDSI/ArachneCommons/tree/synapse to be merged to https://github.com/OHDSI/ArachneCommons/tree/1.17.x and released after)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -29,7 +29,7 @@
 
     <circe.version>1.9.4</circe.version>
     <jersey.version>2.14</jersey.version>
-    <SqlRender.version>1.9.0</SqlRender.version>
+    <SqlRender.version>1.9.444-SNAPSHOT</SqlRender.version>
```
Comment (anthonysena) [inline pom.xml:32]: Please use the recently released v1.10.0 of SqlRender.

**Review by anthonysena — CHANGES_REQUESTED** [review]

File: pom.xml [hunk 2]
```diff
@@ -193,7 +193,7 @@
     <server.ssl.key-password></server.ssl.key-password>
     <server.servlet.context-path>/WebAPI</server.servlet.context-path>
 
-    <arachne.version>1.17.2</arachne.version>
+    <arachne.version>1.17.444-SNAPSHOT</arachne.version>
```
Comment (alex-odysseus) [inline pom.xml:196]: 1.17.3

**Comment by chrisknoll** [comment]
Also, please merge with master and resolve conflicts.

**Comment by alex-odysseus** [comment]
Closing as been replaced by https://github.com/OHDSI/WebAPI/pull/2097

**Pull request closed without merging**